metpy/calc/kinematics.py

```diff
@@ -748,3 +748,48 @@ def absolute_vorticity(u, v, dx, dy, lats, dim_order='yx'):
     f = coriolis_parameter(lats)
     relative_vorticity = vorticity(u, v, dx, dy, dim_order=dim_order)
     return relative_vorticity + f
+
+@exporter.export
+@check_units('[speed]', '[speed]', '[length]', '[length]')
+def potential_vorticity(u, v, dx, dy, lats, potential_temperature, pressure,
+                        axis=0, dim_order='yx'):
+    """Calculate the potential vorticity in a layer.
+
+        Parameters
+        ----------
+        u : (M, N) ndarray
+            x component of the wind
+        v : (M, N) ndarray
+            y component of the wind
+        dx : float
+            The grid spacing in the x-direction
+        dy : float
+            The grid spacing in the y-direction
+        lats : (M, N) ndarray
+            latitudes of the wind data
+        potential_temperature : (M, N, O) ndarray
```

Comment:
We need to specify that these need to be three levels of data, but I don't like (M, N, 3) or (3, M, N) as the `axis` argument makes the ordering unimportant.